### DISCUSSION TITLE
PackageId empty on Get-WtWin32Apps

### DISCUSSION THREAD

**Discussion by verinco-davy** [svrooij/WinTuner#71] [Q&A]
Getting empty packageID's; anyone got the same?

<img width="679" alt="2024-05-22 19_44_43-New Discussion · svrooij_WingetIntune" src="https://github.com/svrooij/WingetIntune/assets/83293240/7979a733-0883-4977-b4f1-4b63a590b055">

**Comment by svrooij** [comment]
You're not the first one, which is why I fixed this in version [0.8.6](https://github.com/svrooij/WingetIntune/discussions/68)
